src/style.css

```
@@ -1,3 +1,103 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  height: 100%;
+  margin-top: 0;
+}
+
+.nav {
+  padding: 0;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  font-size: 12px;
+  text-align: center;
+}
+
+.nav__item {
+  margin-left: 20px;
+  font-size: 12px;
+  text-align: right;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-style: normal;
+  line-height: 60px;
+  font-weight: 500;
+  text-align: right;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link:hover {
+  color: var(--link-color);
+  border-radius: 8px;
+  height: 4px;
+  max-height: 60px;
+}
+
+.logo_link {
+  display: block;
+  height: 40px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.is-active {
+  color: var(--link-color);
+  border-radius: 8px;
+  height: 4px;
+  max-height: 60px;
+  bottom: 0;
+  padding-bottom: 10px;
+}
```

no needs to duplicate rules from `nav__link`
```suggestion
.is-active {
  color: var(--link-color);
  position: relative;
}
```